int32 motor_kiri
int32 motor_kanan
int32 servo_kiri
int32 servo_kanan
int32 tekin_kiri
int32 tekin_kanan